src/FakeItEasy/Core/FakeManager.ObjectMemberRule.cs

```diff
@@ -1,6 +1,6 @@
 namespace FakeItEasy.Core
 {
-    using static FakeItEasy.ObjectMembers;
+    using static FakeItEasy.ObjectMethod;

```

Comment:
I had no idea you could do that on enums!

Reply:
Yes, but it was a class, not an enum

Reply:
I am not the shiniest penny in the fountain.